classes/class-wc-connect-help-provider.php

```diff
@@ -2,20 +2,263 @@
 
 if ( ! class_exists( 'WC_Connect_Help_Provider' ) ) {
 
+	define( 'WOOCOMMERCE_CONNECT_MINIMUM_WOOCOMMERCE_VERSION', '2.6' );
+	define( 'WOOCOMMERCE_CONNECT_MINIMUM_JETPACK_VERSION', '4.0.2' );
+	define( 'WOOCOMMERCE_CONNECT_MAXIMUM_SCHEMA_AGE_SECONDS', 3 * DAY_IN_SECONDS );
+
 	class WC_Connect_Help_Provider {
 
 		/**
+		 * @var WC_Connect_Service_Schemas_Store
+		 */
+		protected $service_schemas_store;
+
+		/**
 		 * @var WC_Connect_Service_Settings_Store
 		 */
 		protected $service_settings_store;
 
-		public function __construct( WC_Connect_Service_Settings_Store $service_settings_store ) {
+		/**
+		 * @array
+		 */
+		protected $fieldsets;
+
+		public function __construct( WC_Connect_Service_Schemas_Store $service_schemas_store, WC_Connect_Service_Settings_Store $service_settings_store ) {
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-connect-client/pull/349/commits/a7353cea96a89ad17253944326fb8c299a88c9b3